### DISCUSSION TITLE
Can't start docker container on windows

### DISCUSSION THREAD

**Discussion by Fosterefe** [monkeytypegame/monkeytype#4108] [Questions]
I tried to run monkeytype using docker, I followed the steps but i always get this error when i run the container: 

![image](https://user-images.githubusercontent.com/90114082/227390265-0b512ce3-31d3-466c-ba64-7551aec4d213.png)

Does someone have an explanationm to this?

**Comment by Miodec** [comment]
![image](https://user-images.githubusercontent.com/13181393/227483644-2c624fb8-70ad-459a-b46b-570d17ee5937.png)
Make sure to install all dependencies.
